ImagineAR/IAR-SDK-Sample-iOS
# IAR-5354 Add the Location command to Core sample app

Overview

- Add Location command as a custom command for debug tools

Resources

- IAR-5354

Testing Instructions

- Open the Debug tool, enable simulated location, add a value and see if it's the same when opening Location Markers
